### ISSUE TITLE
bug when starting point is a full array or tensor

### ISSUE THREAD

**Issue by Hua-Zhou** [Hua-Zhou/TensorReg#1]
Reported by Xiwei Tang. 
```matlab
kruskal_reg(X, M, y, r, 'normal', 'B0', Bint)
```
report error 
```matlab
Error using horzcat
Dimensions of matrices being concatenated are not consistent.
```
if Bint is supplied as a full array or a full tensor.

**Comment by Hua-Zhou** [comment]
The reason is that internally the `kruskal_reg` function first does a CP decomposition of `Bint` which outputs a `ktensor` with `U` being a `d x 1` cell array, while the algorithm expects a `1 x d` cell array.

**Comment by Hua-Zhou** [comment]
Fixed in commit 50d16a6fe6f0810e2f7eb21f50e8e015fb11c206 and tag v1.0.2
